schemas/workflow_runs/definitions.yaml

```diff
@@ -21,6 +21,9 @@ workflow_run_shared:
         type: string
         minLength: 1
         maxLength: 128
+    sdk_token:
+      type: ["null", string]
```

Comment:
Please use the nullable construct instead:
```
# Correct
type: integer
nullable: true
# Incorrect
type: null
# Incorrect as well
type:
  - integer
  - null
```

https://swagger.io/docs/specification/data-models/data-types/